test/kpod_load.bats

```diff
@@ -2,79 +2,62 @@
 
 load helpers
 
-IMAGE="alpine:latest"
+function setup() {
+    copy_images
+}
 
 function teardown() {
     cleanup_test
 }
 
 @test "kpod load input flag" {
-	run ${KPOD_BINARY} ${KPOD_OPTIONS} pull $IMAGE
-	echo "$output"
-	[ "$status" -eq 0 ]
-	run ${KPOD_BINARY} ${KPOD_OPTIONS} save -o alpine.tar $IMAGE
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} save -o alpine.tar $ALPINE
 	echo "$output"
 	[ "$status" -eq 0 ]
-	run ${KPOD_BINARY} ${KPOD_OPTIONS} rmi $IMAGE
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} rmi $ALPINE
 	echo "$output"
 	[ "$status" -eq 0 ]
 	run ${KPOD_BINARY} ${KPOD_OPTIONS} load -i alpine.tar
 	echo "$output"
 	[ "$status" -eq 0 ]
 	rm -f alpine.tar
-	run ${KPOD_BINARY} ${KPOD_OPTIONS} rmi $IMAGE
-	echo "$output"
-	[ "$status" -eq 0 ]
 }
 
 @test "kpod load oci-archive image" {
-	run ${KPOD_BINARY} ${KPOD_OPTIONS} pull $IMAGE
-	[ "$status" -eq 0 ]
-	run ${KPOD_BINARY} ${KPOD_OPTIONS} save -o alpine.tar --format oci-archive $IMAGE
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} save -o alpine.tar --format oci-archive $ALPINE
 	[ "$status" -eq 0 ]
-	run ${KPOD_BINARY} $KPOD_OPTIONS rmi $IMAGE
+	run ${KPOD_BINARY} $KPOD_OPTIONS rmi $ALPINE
 	[ "$status" -eq 0 ]
 	run ${KPOD_BINARY} ${KPOD_OPTIONS} load -i alpine.tar
 	echo "$output"
 	[ "$status" -eq 0 ]
 	rm -f alpine.tar
-	run ${KPOD_BINARY} $KPOD_OPTIONS rmi $IMAGE
-	[ "$status" -eq 0 ]
 }
 
 @test "kpod load oci-archive image with signature-policy" {
-	run ${KPOD_BINARY} ${KPOD_OPTIONS} pull $IMAGE
-	[ "$status" -eq 0 ]
-	run ${KPOD_BINARY} ${KPOD_OPTIONS} save -o alpine.tar --format oci-archive $IMAGE
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} save -o alpine.tar --format oci-archive $ALPINE
 	[ "$status" -eq 0 ]
-	run ${KPOD_BINARY} $KPOD_OPTIONS rmi $IMAGE
+	run ${KPOD_BINARY} $KPOD_OPTIONS rmi $ALPINE
 	[ "$status" -eq 0 ]
 	cp /etc/containers/policy.json /tmp
```

Comment:
Shouldn't this be the `policy.json` from the source directory's `test` subdirectory?